### PR TITLE
Thchang/2138 disable spell check

### LINKED ISSUE #2138: disable spell check in text input fields
**Describe the bug**
In the text input field like
Screen Shot 2023-04-07 at 20 14 07
spell check will be executed so if the input text is considered as "wrong" we see the red highlights. We should try to disable spell check in the text input field elsewhere in CARTA.

We can apply spellCheck=false to <input>

**Expected behavior**
No spell check is applied to text input fields.

**Screenshots or videos**
If applicable, add screenshots/videos to help explain your problem.

**Platform info (please complete the following information):**
 - OS [e.g. macOS Monterey]: macOS Monterey
 - Browser [e.g. chrome, safari, electron app]: v3-stable electron app, dev
 - Browser version [e.g. 22]: 112
 - Backend branch [e.g. dev, v3b2 release]: v3-stable, dev
 - Frontend branch [e.g. dev, v3b2 release]: v3-stable, dev

**Additional context**

### PR DESCRIPTION
**Pull request by TienHao**
**Description**

Closed #2138 by adding `spellcheck="false"` in text input.

* Adding spellcheck in text input without `spellcheck="false"`.
* In case any developer forget to disable spell check, added an event listener to add `spellcheck="false"` to the text input once an input event is triggered.

**Checklist**

For linked issues (if there are):
- [x] assignee and label added
- [x] ZenHub issue connection, board status, and estimate updated

For the pull request:
- [x] reviewers and assignee added
- [x] ZenHub estimate, milestone, and release (if needed) added
- [x] e2e test passing / ~corresponding fix added~
- [x] changelog updated / ~no changelog update needed~
- [x] ~protobuf updated to the latest dev commit~ / no protobuf update needed
- [x] `BackendService` unchanged / ~`BackendService` changed and corresponding ICD test fix added~

### CODE REVIEW

**Review by kswang1029 — CHANGES_REQUESTED** [review]
@TienHao this mostly looks good. Please enable spell check for the text annotation.
A question: for existing input, we manually disable spell check. If we add a new input in the future do you mean we don't have to disable it manually based on the new event listener in AppStore?

**Comment by TienHao** [comment]
I have enabled spell check for the text annotation. (no more spellcheck changes for textarea elements in the new event listener)

>  A question: for existing input, we manually disable spell check. If we add a new input in the future do you mean we don't have to disable it manually based on the new event listener in AppStore?

Yes, we don't need to disable it manually. I made a few change in the event listener. The default value of spellcheck is false now, and if some text input still need a spell check just adding `spellCheck={true}` in them.

**Review by kswang1029 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 * Changed the default title string in the image viewer ([#2168](https://github.com/CARTAvis/carta-frontend/issues/2168)).
 * Modified text annotation textbox to stay the same dimension as user zoom the image ([#2162](https://github.com/CARTAvis/carta-frontend/issues/2162)).
+* Disabled spell check in text input and textarea ([#2138](https://github.com/CARTAvis/carta-frontend/issues/2138)).
```
Comment (YuHsuan-Hwang) [inline CHANGELOG.md]: Remove "textarea."

**Review by YuHsuan-Hwang — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (TienHao) [reply CHANGELOG.md]: thanks, just fixed!

**Review by crocka — APPROVED** [review]

**Pull request merged**